urlscan/help.md

```diff
@@ -20,9 +20,19 @@ on potential malicous indicators.
 
 The connection configuration accepts the following parameters:
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|api_key|credential_secret_key|None|False|urlscan API key. Not required for the search action|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|api_key|credential_secret_key|None|False|urlscan API key. Not required for the search action|None|a46ec67a-0f2e7-c3879-26ac5d783ea4b8|
+
+Example input:
+
+```
+{
+  "api_key": {
+    "secretKey": “9de5069-c5afe602b-2ea0a04b66beb2c0”
```

Comment:
```suggestion
    "secretKey": “a46ec67a-0f2e7-c3879-26ac5d783ea4b8”
```

Reply:
We should keep these the same. We will auto-generate the raw JSON as well from the example key soon.